clients/java/signalr/src/main/java/com/microsoft/signalr/HubConnection.java

```diff
@@ -55,18 +55,38 @@
     private final Logger logger = LoggerFactory.getLogger(HubConnection.class);
 
 
+    /**
+     * Sets the server timeout interval for the connection.
+     *
+     * @param serverTimeout The timeout duration.
```

Comment:
The **server** timeout duration

(so it agrees with `getServerTimeout`'s `@return`)